example/getusersecret.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+if [ $(which wget | wc -l) -eq 0 ]; then echo 'Missing "wget". Aborting'; exit 1; fi
+if [ $(which jq | wc -l) -eq 0 ]; then echo 'Missing "jq". Aborting'; exit 1; fi
+if [ $(which base64 | wc -l) -eq 0 ]; then echo 'Missing "base64". Aborting'; exit 1; fi
+if [ $(which xz | wc -l) -eq 0 ]; then echo 'Missing "xz". Aborting'; exit 1; fi
+
+SERVER="acme-staging.api.letsencrypt.org"
+EMAIL="test@test.com"
+
+while [[ $# > 1 ]]
+do
+  key="$1"
+  case $key in
+    -e|--email) EMAIL="$2"; shift ;;
+    -s|--server) SERVER="$2"; shift ;;
+  esac
+  shift
+done
+
+echo
+echo "### Ready to register user with $EMAIL on $SERVER (Hit any key to proceed or Ctrl+C to cancel)"
+echo
+read
+echo "Processing..."
+
+TMPDIR="$(pwd)/.k8s-lego-tmp/"
+rm -rf $TMPDIR
+
+wget https://github.com/xenolf/lego/releases/download/v0.3.0/lego_linux_amd64.tar.xz -c -P $TMPDIR -q
```

Comment:
Needs to be cross-platform somehow (can't require linux - I'd like to also support osx/darwin. Windows is probably fine for now, although they have recently added bash for windows so it'd be great if it can be supported)